Update README.md added slack button (#138)
creativecommons/creativecommons-base: README.md

# Contribute
Contributions are highly appreciated. Please see [`CONTRIBUTING.md`](CONTRIBUTING.md).

## 📣 Let's chat

> Do you want to ask or chat to one of our contributers just click the slack link and let's have a chat.

Reviewer: ```suggestion
Do you want to connect with our contributors?
```

> Just click the button bellow and follow the onscreen instructions.

Reviewer: ```suggestion
Just click the button below and follow the instructions.
```
